virtual-server/examples/python/README.md

```diff
@@ -0,0 +1,28 @@
+# Python example
+
+Python example for talking to a VirtualServer custom controller and kubevirt.
```

Comment:
Language edit: "An example Python implementation of a kubernetes client that interacts with the Coreweave VirtualServer resource as well as the kubevirt subresource api."